pkgs/tools/misc/tmate/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, autoconf, automake110x, libtool, pkgconfig, zlib, openssl, libevent, ncurses, cmake, ruby }:
+
+stdenv.mkDerivation rec {
+  name = "tmate-${version}";
+  version = "1.8.10";
+
+  src = fetchFromGitHub {
+    owner  = "nviennot";
+    repo   = "tmate";
+    rev    = version;
+    sha256 = "1bd9mi8fx40608zlady9dbv21kbdwc3kqrgz012m529f6cbysmzc";
+  };
+
+  buildInputs = [ autoconf automake110x pkgconfig libtool zlib openssl libevent ncurses cmake ruby ];
+
+  # It seems that the project requires cmake to be in the PATH, but
+  # the projet does not use cmake in the configure phase. The
+  # following disables the use of cmake for the configure phase.
+  configurePhase = ''
+    ./autogen.sh
+    configureFlags="--prefix=$prefix"
+    echo configure flags: $configureFlags
+    ./configure $configureFlags
+  '';
```

Comment:
Wouldn't it be better to just define `dontUseCmakeConfigure = true;`? (I didn't test it does exactly what you need.)